mindsdb/integrations/handlers/oracle_handler/oracle_handler.py

```diff
@@ -60,7 +60,8 @@ def __init__(self, name: str, connection_data: Optional[dict], **kwargs):
     def connect(self) -> Connection:
         if self.is_connected is True:
             return self.connection
-
+        d = None  # default suitable for Linux
+        oracledb.init_oracle_client(lib_dir=d)
```

Comment:
@tanbirali Can you please provide more info on how this provides the thick mode support? Is this working

Reply:
https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enabling-python-oracledb-thick-mode
as per this docs this should pretty much enable the thick mode in linux system.